ping.go

```diff
@@ -6,9 +6,9 @@ import (
 
 type HealthModule struct{}
 
-func (m HealthModule) Register(e *gin.Engine) {
-	e.GET("/", m.ping)
-	e.GET("/health", m.health)
+func (m HealthModule) Register(g *gin.RouterGroup) {
+	g.GET("/ping", m.ping)
```

Comment:
Still having an endpoint listening on GET requests on the root path is nice just so that it's easier to debug when setting things up. If you start up the wharf-api and then go to its IP and port via the web browser, having something respond instead of the 404 error from the browser is very helpful.